### PR TITLE
Agpush 1430.use.plist.for.config.objc

### PR DESCRIPTION
**Pull request by corinnekrych**
Objc version of plist configuration. To be merged in 1.x_dev branch.
@sebastienblanc mind to review?

### CODE REVIEW

**Comment by matzew** [comment]
Is that a separate PList file? can that info be applied (easily) on an existing? 

Is PList because that's more natural/simpler than JSON?

**Comment by corinnekrych** [comment]
@matzew plist are the default config file for ios, it sounds more native to use them. we discussed the point on this thread: http://aerogear-dev.1069024.n5.nabble.com/aerogear-dev-cordova-Push-Plugin-externalise-Push-config-td9361.html
it uses the default plist file associated to the application. Not a separate one.

**Comment by matzew** [comment]
ok, great! should we update the README and/or the doc for this change/improvement ?

**Comment by corinnekrych** [comment]
@matzew good point. I've just added a section in README.

File: push-sdk/AGDeviceRegistration.h [hunk 1]
```diff
@@ -38,8 +38,28 @@ extern NSString * const AGNetworkingOperationFailingURLResponseErrorKey;
...
- * start receiving messages.
+ * An initializer method to instantiate an AGDeviceRegistration object.
+ *
+ * @return the AGDeviceRegistration object.
+ */
+- (id) init;
+
+
+/**
+ * Registers your mobile device to the AeroGear Push server so it can start receiving messages.
```
Comment (sebastienblanc) [inline push-sdk/AGDeviceRegistration.h]: s/Push/UnifiedPush

File: push-sdk/AGDeviceRegistration.h [hunk 2]
```diff
@@ -38,8 +38,28 @@ extern NSString * const AGNetworkingOperationFailingURLResponseErrorKey;
...
+ * Registration information can be provided within clientInfo block or by providin a plist file
```
Comment (sebastienblanc) [inline push-sdk/AGDeviceRegistration.h]: `providing`

**Comment by corinnekrych** [comment]
@sebastienblanc thanks for the typos. I fixed them in both PR (swift objc)
you can test it on helloword with https://github.com/jboss-mobile/unified-push-helloworld/pull/12
ready to merge?

**Pull request merged**